pypeerassets/__main__.py

```diff
@@ -161,8 +161,9 @@ def find_card_bundles(provider: Provider, deck: Deck) -> Optional[Iterator]:
         if deck.id is None:
             raise Exception("deck.id required to listtransactions")
 
+        p2th_account = provider.getaccount(deck.p2th_address)
         batch_data = [('getrawtransaction', [i["txid"], 1]) for
-                      i in provider.listtransactions(deck.id)]
+                      i in provider.listtransactions(p2th_account)]
```

Comment:
I'm assuming pypa is no longer using rpcnode accounts and now uses txindex by default. If so this is great.

Reply:
@saeveritt I am not sure what do you mean. Rpc integration in pypa is your work. I've never used it in my personal tests and setup as I consider local RpcNode a bad backend for the PA.

Reply:
You're focused on using block explorers as the main data-source, correct? Any solutions in the future involving a lite client for pa or backends for pa specific block explorers will still use the rpcnode to gather data. In production when a deck grows to a certain size it will be a bottleneck to rely on standard blockexplorers then parse the data as it comes in and calculate states. The logic in maturing the rpc interaction is for future use by a backend PA API.